### DISCUSSION TITLE
New install presents me with a grub prompt

### DISCUSSION THREAD

**Discussion by Pyrasanth** [Security-Onion-Solutions/securityonion#2271] [Unsupported Versions]
Hi, 

I have been trying to install the latest iso to no avail.  It starts, displays a few lines and then after about 4 seconds it displays a grub prompt.  I have been using Rufus to make the bootable image and have tried 3 usb sticks.  I have tried Etcher but the resulting image isn't picked up as a boot option in the BIOS.

I have made sure the image has the right checksum and I can install Ubuntu using the same process with no issues.  I have made sure secure boot is off as I read that can have some impact (can not remember where). Has anyone else had a similar issue or heard of this before?  I am now trying UNetbootIn to see if that makes a difference.

--------------------

Quick follow up - making the image with UNetbootIn had the same result.

**Comment by weslambert** [comment]
Out of curiosity, are you able to install Centos 7 minimal without issue?  The ISO image is based off of Centos, so I'm curious if you would see the same issue.
